spark-indexer/src/main/java/org/uniprot/store/spark/indexer/chebi/ChebiFileMapper.java

```diff
@@ -29,7 +30,8 @@
             return new Tuple2<>(chebi.getId(), chebi);
         } else {
             log.info("ERROR PARSING ChebiFileMapper WITH LINES: " + chebiLines);
-            throw new RuntimeException("ERROR PARSING ChebiFileMapper WITH LINES: " + chebiLines);
+            throw new SparkIndexException(
```

Comment:
In logging, we should use {} type of value replacement instead of strin concat.
Error should be at error level.